### PR TITLE
Add language selector

### PR DESCRIPTION
**Pull request by michaeljqzq**

### CODE REVIEW

File: src/component/language/language.js [hunk 1]
```diff
@@ -0,0 +1,50 @@
...
+        this.setState((prev) => {
+            return {
+                showList: !prev.showList
+            }
+        });
+        e.stopPropagation();
+        this.props.reloadMain();
+    }
+
+    render() {
```
Comment (formulahendry) [inline src/component/language/language.js]: After language is changed, will the url changed? Something like; https://docs.microsoft.com/en-us/azure/iot-hub/iot-hub-node-node-getstarted for en-us and https://docs.microsoft.com/zh-cn/azure/iot-hub/iot-hub-node-node-getstarted for zh-cn.

This has two benefit:
* Better SEO for certain language
* Good for sharing: e.g. A Chinese could simply share the Chinese content to his friend.

Comment (michaeljqzq) [inline src/component/language/language.js]: Good suggestion. I will sync with Xin about SEO problem.

File: src/lib/telemetry.js [hunk 2]
```diff
@@ -40,13 +41,13 @@ function getUserId() {
 }
 
 function tracePageViewAI(name, property, metric) {
-  property = Object.assign(property || {referrer:document.referrer}, userProperties);
+  property = Object.assign({referrer:document.referrer,lang:Localization.getLocalizedString().getLanguage()}, property, userProperties);
```
Comment (SLdragon) [inline src/lib/telemetry.js]: Add space after comma

File: src/index.js [hunk 3]
```diff
@@ -126,6 +126,7 @@ class Index extends Component {
     return (
       <div className='main'>
         <Banner 
+        reloadMain={this.forceUpdate.bind(this)}
         toggleHelpState = {this.toggleHelpState} />
         {
           1 === 0 ? (<Toolbar onRunApp={this.runApp} />) : ('')
```
Comment (SLdragon) [inline src/index.js]: Simplify this expression

File: src/lib/telemetry.js (hunk 2 above)
Comment (michaeljqzq) [inline src/lib/telemetry.js]: fixed

File: src/index.js (hunk 3 above)
Comment (michaeljqzq) [inline src/index.js]: fixed

**Pull request merged**